lib/pengine/pe_output.c

```diff
@@ -2682,8 +2682,8 @@ resource_list(pcmk__output_t *out, va_list args)
         int x;
 
         /* Complex resources may have some sub-resources active and some inactive */
-        gboolean is_active = rsc->fns->active(rsc, TRUE);
-        gboolean partially_active = rsc->fns->active(rsc, FALSE);
+        bool is_active = rsc->fns->active(rsc, TRUE);
+        bool partially_active = rsc->fns->active(rsc, FALSE);
```

Comment:
This return value is defined as gboolean in pe_types.h (unfortunately public API)